pegasus/sites.v3/code.org/public/css/ai-skinny-banner-2023.scss

```diff
@@ -0,0 +1,109 @@
+@import "color", "font", "breakpoints";
+
+.ai-skinny-banner-2023-default {
+  box-sizing: content-box;
+  height: 130px;
+  background: url("/images/banners/ai-bg.png") top center no-repeat;
+  background-size: cover;
+
+  @media screen and (max-width: $width-md) {
+    padding: 0 1.5em;
+  }
+
+  @media screen and (max-width: $width-sm) {
+    padding: 1em 1em 0.5em;
+    height: unset;
+  }
+
+  a:hover,
+  a:active,
+  a:visited,
+  a:link {
+    text-decoration: none;
+  }
+
+  .wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    max-width: 960px;
+    margin: 0 auto;
+
+    @media screen and (max-width: $width-sm) {
+      justify-content: center;
+    }
+  }
+
+  img {
+    margin: 0 2em;
+
+    @media screen and (max-width: $width-md) {
+      margin: 0;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      width: 50px;
+      margin-right: 1.5em;
+    }
+
+    @media screen and (max-width: $width-xs) {
+      margin: 0 0 1em 0;
+    }
+  }
+
+  .text-wrapper {
+    color: white;
+    text-align: center;
+
+    @media screen and (max-width: $width-md) {
+      text-align: left;
+      width: 57%;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      width: 75%;
+    }
+
+    @media screen and (max-width: $width-xs) {
+      width: 100%;
+      text-align: center;
```

Comment:
nit –– can you leave off `text-align` since it's defined on 57?

Reply:
This is overriding the `text-align: left` for tablet and mobile since it needs to switch to center again on extra small mobiles screens — this screenshot is `320px`:

<img width="342" alt="Screenshot 2023-04-24 at 2 39 41 PM" src="https://user-images.githubusercontent.com/9256643/234122692-1e4cab9f-306e-413f-975a-5c452f0ccfdd.png">

Reply:
Ah I see –– that makes sense!